tests/test_codebase/test_mmdet3d/test_mmdet3d_models.py

```diff
@@ -144,9 +146,12 @@ def test_centerpoint(backend_type: Backend):
                 output_names=['outputs']),
             codebase_config=dict(
                 type=Codebase.MMDET3D.value, task=Task.VOXEL_DETECTION.value)))
-    model_cfg = load_config('data/model_cfg.py')[0]
+    model_cfg = load_config(
+        'tests/test_codebase/test_mmdet3d/data/model_cfg.pytests/'
```

Comment:
could you create a constant global variable representing this model config path?